pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -315,13 +336,13 @@ def connection_list(context):
     """
     List the WBEM connection definitions.
 
-    List the WBEM connection definitions in the connections file as a table.
-
-    In the table, the current connection is marked with an "*" after its name.
+    This command displays all entries in the connections file and the
+    current connection if it is not in the connections file as
```

Comment:
A current connection is not always defined, so the sentence should allow for that case as well.

Reply:
Hmmm. That one was from old version.  Now reads

    An "!" before the name indicates the selected connection.
    A '*' before the name indicates that it is the current connection.

Reply:
The response does not address the comment.

Reply:
DONE. Added if it exists